src/main/java/com/server/RequestReader.java

```diff
@@ -3,27 +3,31 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 
+import static java.lang.Integer.parseInt;
+
 public class RequestReader {
 
     private BufferedReader bufferedReader;
+    private int contentLength;
 
     RequestReader(BufferedReader bufferedReader) throws IOException {
         this.bufferedReader = bufferedReader;
     }
 
     public String getHeader() throws IOException {
-        return readHeader();
+        String header = read("\r\n");
```

Comment:
This method is called getHeader but it looks like the method is getting the body too under certain conditions. Should we perhaps have another method that gets the body so that getHeader only "gets the header"?